src/mrinufft/io/siemens.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+
+try:
+    from mapvbvd import mapVBVD
```

Comment:
This needs to also be added to `pyproject.toml` (maybe in an `io` dependency bundle ?)